_1327/main/views.py

```diff
@@ -30,8 +31,12 @@ def index(request):
 		document = Document.objects.get(id=settings.MAIN_PAGE_ID)
 		return HttpResponseRedirect(reverse('documents:view', args=[document.url_title]))
 
-		md = markdown.Markdown(safe_mode='escape', extensions=[TocExtension(baselevel=2)])
-		text = md.convert(document.text)
+		md = markdown.Markdown(safe_mode='escape', extensions=[TocExtension(baselevel=2), 'markdown.extensions.abbr'])
+		abbrs = AbbreviationExplanation.objects.all()
+		abbr_list = ""
```

Comment:
here again - please use a function that handles this somewhere and use it in all 5 places